### PR TITLE
 Last Comment Bug 1172624 - Convert python marionette test_gallery_empty.py script to marionettejs test

### PR DESCRIPTION
**Pull request by nojunpark**
…to marionettejs test

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=013e78eda07ba15b761e695cfed4abbb69768525

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5149b990b34fe348a2f8532126cff0940a5f5d54

File: apps/gallery/test/marionette/empty_image_test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  var app, client;
+
+  client = marionette.client({
+    profile: {
+      prefs: {
+        'device.storage.enabled': true,
+        'device.storage.testing': true,
+        'device.storage.prompt.testing': true,
+        'webgl.force-enabled': true
```
Comment (punamdahiya) [inline apps/gallery/test/marionette/empty_image_test.js]: Please check and include settings that are must have, I think webgl.force-enabled is not needed for this test

File: apps/gallery/test/marionette/lib/gallery.js [hunk 2]
```diff
@@ -92,6 +95,28 @@ Gallery.prototype = {
...
+  
+  /**
+   * @return {Marionette.Element} Container for overlay message content.
+   */
+  get overlayText() {
+    return this.client.helper.waitForElement(Gallery.Selector.overlayText);
+  },
+
+  /**
+   * @return {Marionette.Element} Container for overlay message content.
```
Comment (punamdahiya) [inline apps/gallery/test/marionette/lib/gallery.js]: Please fix the comment to refer camera button

File: apps/gallery/test/marionette/manifest.ini [hunk 3]
```diff
@@ -1,3 +1,4 @@
 [edit_image_test.js]
 [fullscreen_image_test.js]
 [delete_image_test.js]
+[empty_image_test.js]
```
Comment (punamdahiya) [inline apps/gallery/test/marionette/manifest.ini]: I don't think we need to update manifest.ini file. See https://bugzilla.mozilla.org/show_bug.cgi?id=1072582 for bug to delete these files. https://bugzilla.mozilla.org/show_bug.cgi?id=1039140#c19 indicating that it's not used in production

File: apps/gallery/test/marionette/empty_image_test.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    client.helper.waitForElement(overlayView);
+    assert.ok(overlayView.displayed());
+    assert.ok(overlayTitle.displayed());
+    assert.ok(overlayTitle.text() == 'No photos or videos');
+    assert.ok(overlayText.displayed());
+    assert.ok(overlayText.text() == 'Use the Camera app to get started.');
+    assert.ok(cameraButton.displayed());
+    assert.ok(cameraButton.text() == 'Go to Camera');
+  });
+});
```
Comment (punamdahiya) [inline apps/gallery/test/marionette/empty_image_test.js]: Add New line

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b1d630abe2a24f848cacf074cc0ef98d6bc2c9de

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c1dc82b10ab3017dc040c75e1220b8b20fa5969b

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d288f8628fb94059bd3caf50aa95192c9a49542f

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=65fb916bc4c39afe07b4241f80072d1d665ebc5a

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=70c513fa6e9c005a6d8321bf948891924ae0e8be

**Pull request merged**